internal/tasks/fixtures/clair/manifest-004-failure.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file has the same contents as `internal/tasks/fixtures/clair/manifest-004.json` and can be deduplicated.